src/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { Theme, styled } from '@mui/material/styles';
+import MuiStepper from '@mui/material/Stepper';
+import Step from '@mui/material/Step';
+import StepLabel from '@mui/material/StepLabel';
+import NavigateNext from '@mui/icons-material/NavigateNext';
+import { StepIconProps } from '@mui/material/StepIcon';
+
+const CustomStep = styled(Step)(({ theme }) => ({
+  padding: theme.spacing(1),
+}));
+
+const stepStateStyles = (theme: Theme) => ({
+  ...theme.typography.headline_ss_xxs,
+  color: `${theme.palette.grey['500']}`,
+  '& .MuiStepLabel-label.Mui-active': {
+    color: theme.palette.secondary.main,
+  },
+  '& .MuiStepLabel-label.Mui-completed': {
+    color: theme.palette.primary.light,
+  },
+});
+
+const CustomStepLabel = styled(StepLabel)(({ theme }) => ({
+  ...stepStateStyles(theme),
+  cursor: 'pointer !important',
+}));
+
+const Connector = () => {
+  return (
+    <StepLabel
+      sx={(theme) => ({
+        '& .MuiStepLabel-label': {
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+          color: theme.palette.grey['500'],
+        },
+        ...stepStateStyles(theme),
+      })}
+      icon={<span />}
+    >
+      <NavigateNext fontSize="small" />
+    </StepLabel>
+  );
+};
+
+const CustomStepIconRoot = styled('div')<{
+  ownerState: { completed?: boolean; active?: boolean };
+}>(({ theme, ownerState }) => ({
+  color: theme.palette.grey['500'],
+  ...theme.typography.headline_ss_xxs,
+  ...(ownerState.active && {
+    color: theme.palette.secondary.main,
+  }),
+  ...(ownerState.completed && {
+    color: theme.palette.primary.light,
+  }),
+}));
+
+function CustomStepConnector(props: StepIconProps) {
+  const { active, completed, className } = props;
+
+  return (
+    <CustomStepIconRoot ownerState={{ completed, active }} className={className}>
+      {String(props.icon)}.
+    </CustomStepIconRoot>
+  );
+}
```

Comment:
@BrianJunk what do you think about this?

I have created a custom step connector which displays as `1. ...` `2. ...`
Another approach is adding the count indicator in the steps arrays string

so you can pass `steps = ['1. Default', '2. Others']`

Another approach also is on line 83
Adding `{index+1}. {label}`

Reply:
leave it like this for now, we can add the steps later if needed